src/main/resources/staff/templates/document/countyCourtJudgmentDetails.html

```diff
@@ -239,12 +243,33 @@ <h2>Request for judgment by default</h2>
           {{ ccj.amount.remainingAmount }}
         </td>
       </tr>
-      <tr>
-        <td>When you want the defendant to pay:</td>
-        <td>
-          {{ ccj.repaymentOption }}
-        </td>
-      </tr>
+      {% if ccj.repaymentOption is not empty %}
+        <tr>
+          <td>How do you want the defendant to pay:</td>
+          <td>
+            {{ ccj.paymentOption }}
```

Comment:
That option is independent from repayment plan, don't we want do display it for all payment options like:

```
IMMEDIATELY,
FULL_BY_SPECIFIED_DATE,
INSTALMENTS;
```